### DISCUSSION TITLE
It is possible to modify a newly created record from inside an onModelAfterCreate hook ?

### DISCUSSION THREAD

**Discussion by simboden** [pocketbase/pocketbase#5031] [Q&A] [answered]
I will need this feature, but it seems not supported. Any information will be useful, Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
It is supported but since it is an `after` hook you need to persist it manually. For updating records you can check:

- Go: https://pocketbase.io/docs/go-records/#update-existing-record
- JSVM: https://pocketbase.io/docs/js-records/#update-existing-record

If the above doesn't help, please provide more details about your use case and the desired behavior.

**Reply by simboden** [reply]
Thank you -- I finally got it, and I confirm that it works --- I just want to hilight that 
e.model.set( ... ) ; $app.dao().saveRecord( e.model ); work as expected, 
but if you retrieve the same record using a query then saveRecord will fail. 
This was the things that confused me at first. Thanks again
